assets/js/modules/analytics/datastore/accounts.test.js

```diff
@@ -132,7 +119,8 @@ describe( 'modules/analytics accounts', () => {
 		} );
 
 		describe( 'resetAccounts', () => {
-			it( 'sets accounts and related values back to their initial values', () => {
+			it( 'sets accounts and related values back to their initial values', async () => {
+				fetchMock.get( { query: { tagverify: '1' } }, { body: {}, status: 200 } );
```

Comment:
This should be a `receiveExistingTag( null )` instead of a fetch mock.

Reply:
:+1: